pages/star-search/index.tsx

```diff
@@ -669,16 +670,21 @@ function SuggestionBoxes({
       } gap-2 lg:gap-4 w-full max-w-3xl`}
     >
       {suggestions.map((suggestion, i) => (
-        <button key={i} onClick={() => addPromptInput(suggestion.prompt)}>
-          <Card
-            className={`${
-              isHorizontal ? "w-[30rem] snap-start" : "w-full h-fit"
-            } shadow-md border-none text-start !p-6 text-slate-600`}
+        <Card key={i} className="!p-6 shadow-md border-none text-slate-600">
+          <button
```

Comment:
The buttons for suggestions didn't have labels. The style of the suggestions is still the same, but now they are more accessible.